doc/src/config-pg.md

```diff
@@ -1,49 +1,160 @@
-# PostgreSQL configuration
+# PostgreSQL Configuration for PL/Rust
 
-PL/Rust has a two **required** configuration options and a variety of non-required options.
-PL/Rust **must** be configured as a `shared_preload_libraries` entry in `postgresql.conf`.
-PL/Rust also requires `plrust.work_dir` to save intermediate files.
+PL/Rust has two **required** configuration options and a variety of non-required options.
+These options are set in the standard `postgresql.conf` configuration file used
+by PostgreSQL.
 
-```
+PL/Rust has two required configuration options in untrusted and trusted installations.
+Using PL/Rust with cross compilation support has a 3rd required configuration option.
+Failure to set these variables
+will cause `plrust` extension to not function.
+
+
+## Required
+
+
+PL/Rust has two PostgreSQL configuration options that are always required for use,`shared_preload_libraries` and `plrust.work_dir`.
+
+
+#### `shared_preload_libraries` (string)
+
+The [`shared_preload_libraries` entry](https://www.postgresql.org/docs/current/runtime-config-client.html)
+needs to include `plrust`. This is a comma separated list of libraries that
+need to be pre-loaded in order to operate properly.
+
+```bash
 shared_preload_libraries = 'plrust'
+```
+
+
+
+#### `plrust.work_dir` (string)
+
+The `plrust.work_dir` must be set to location for PL/Rust to save
+necessary intermediate files. This path must be writable by the user running
+the PostgreSQL process, typically `postgres` on common Linux distributions.
+
+```bash
 plrust.work_dir = '/tmp'
 ```
 
 
-Failure to set these required variables will cause `plrust` extension to not function.
+## Lints
 
-The PL/Rust-specific configuration options are in the following table.
+#### `plrust.compile_lints` (string)
```

Comment:
We should probably discourage users from tweaking this (or at least note that doing so can completely compromise the "trusted"-ness of plrustc).

I think if they tweak it it also prevents us from updating it in new versions?

Reply:
:+1: Same with `plrust.required_lints`?

Reply:
Yes, required lints are, well, required.

Reply:
Yeah, that makes sense. Thanks!

I moved the lints config section to the end of the page and added a warning to not adjust them.